vendor.mod

```diff
@@ -47,37 +49,10 @@ require (
 )
 
 replace (
-	cloud.google.com/go => cloud.google.com/go v0.44.3
 	github.com/docker/distribution => github.com/docker/distribution v2.7.1-0.20190205005809-0d3efadf0154+incompatible
```

Comment:
maybe this one should be removed from `replace` too? and use latest v2 stable?

Reply:
Yes, I wanted to wait for v2.8.1, then update to that version